src/CFamily.UnitTests/Rules/RulesConfigFixupTests.cs

```diff
@@ -18,28 +18,28 @@
  * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  */
 
+using System.Linq;
 using FluentAssertions;
 using Microsoft.VisualStudio.TestTools.UnitTesting;
-using SonarLint.VisualStudio.CFamily.Rules;
 using SonarLint.VisualStudio.Core;
 using SonarLint.VisualStudio.Integration;
 using SonarLint.VisualStudio.Integration.UnitTests;
 
-namespace SonarLint.VisualStudio.CFamily.UnitTests.Rules
+namespace SonarLint.VisualStudio.CFamily.Rules.UnitTests
 {
     [TestClass]
     public class RulesConfigFixupTests
     {
         [TestMethod]
-        public void Apply_EmptySettings_NoError()
```

Comment:
This test is superceded by the new test `Apply_NoCustomRules_ExcludedRulesAreDisabled` below.